Adyen/Core/Errors/AppleWalletError.swift

```diff
@@ -0,0 +1,14 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+/// An error that occurred during adding a pass to apple wallet.
+public enum AppleWalletError: LocalizedError {
```

Comment:
This could be a `VoucherComponent.Error`

Reply:
yeah, its also used by the `AppleWalletPassProvider` which is in core module.

Reply:
So how about `AppleWalletProvider.Error` ?

Reply:
there is no reason to attach it to `AppleWalletProvider`, its used by more than one component, so thought to put it at a top level enum.